### DISCUSSION TITLE
Get-PnPMicrosoft365Group - HiddenFromAddressListsEnabled property

### DISCUSSION THREAD

**Discussion by Nibushi** [pnp/powershell#1417] [Q&A]
Is it possible to return the HiddenFromAddressListsEnabled property using Get-PnPMicrosoft365Group? 

I know you can use the Set-PnPMicrosoft365Group to set HideFromAddressLists, I was just wondering if it was possible using pnp powershell to check this property first.

Cheers

**Comment by LeonArmston** [comment]
Hi

Yes it is - it's hidden away in the group obbject ResourceBehaviourOptions as HideGroupInOutlook (see image below)

![image](https://user-images.githubusercontent.com/12968962/144826299-e936dd59-b2ee-40cf-b890-473bbc82dc23.png)

**Reply by Nibushi** [reply]
Thanks, I noticed from your screenshot your Group was connected to a Team. So I tried one of my Team connected Groups and I get the same results as you:

![image](https://user-images.githubusercontent.com/6292273/145014848-84e5ebbb-09f8-4ab4-ba7d-a6198a4b262d.png)

However, I notice when using a non-Team connected Group these properties aren't filled in:
![image](https://user-images.githubusercontent.com/6292273/145015041-7b9f194c-3037-4e53-b81e-eb0169d07448.png)

I've since tried a couple of different Groups, some Teams connected and others which aren't and the results seem consistent with what I am showing above. It would be really interesting to know if you see the same behavior.

Not sure if this is an issue or by design. The non-Team connected Groups have the same attributes showing in the Admin UI so I would have hoped these could be surfaced.

Also, I notice that Team connected Groups show as SecurityEnabled where as Groups just connected to a SharePoint Online site show as SecurityEnabled $false. I wonder if that has anything to do with it........down the rabbit hole we go 😊 !

Thanks for your response 👍

**Reply by LeonArmston** [reply]
Hi @Nibushi 

I've done a bit more research into the underlying MS graph API behind the commands and posted in the issue you raised https://github.com/pnp/powershell/issues/1422

Looks like it might be a MS Graph bug

**Reply by Nibushi** [reply]
That's great Leon, thanks for your help
